web-api/src/personalization/routes.rs

```diff
@@ -684,6 +685,9 @@ async fn semantic_search(
     .await?;
 
     if let Some(personalize) = personalize {
+        if matches!(strategy, SearchStrategy::Knn) {
+            normalize_knn_scores(&mut documents);
+        }
```

Comment:
I'm a bit confused.

We ourself move (-1;1) ranges to (0;1) in most situations.

But now we do undo that shift but then in some cases redo it.

Why not always take that scores from ES as we get them, that should not be a bug given the guarantees about values we can give.

Like for personalization we anyway want `(0;1)` general in pretty much all places in the code we assumed we get `(0;1)` ranges from ES instead of "correct (-1;1) ranges".